Source/Microsoft.Teams.Apps.FAQPlusPlus/Bots/FaqPlusPlusBot.cs

```diff
@@ -925,7 +925,18 @@ private static Task<TaskModuleResponse> GetTaskModuleResponseAsync(Attachment qu
                     break;
 
                 default:
-                    this.logger.LogInformation($"Unexpected text in submit payload: {message.Text}", SeverityLevel.Warning);
+                    var payload = ((JObject)message.Value).ToObject<ResponseCardPayload>();
+
+                    if (payload.IsPrompt)
+                    {
+                        this.logger.LogInformation("Sending input to QnAMaker for prompt");
+                        await this.GetQuestionAnswerReplyAsync(turnContext, message).ConfigureAwait(false);
+                    }
+                    else
+                    {
+                        this.logger.LogInformation($"Unexpected text in submit payload: {message.Text}", SeverityLevel.Warning);
```

Comment:
Nit: Use LogWarning and omit SeverityLevel

Reply:
Updated to LogWarning whereever it is required.